tests/Validation/ValidationTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace LanguageServer\Tests;
+
+use Exception;
+use LanguageServer\Definition;
+use LanguageServer\Index\Index;
+use LanguageServer\ParserKind;
+use LanguageServer\PhpDocument;
+use LanguageServer\DefinitionResolver;
+use phpDocumentor\Reflection\DocBlock;
+use phpDocumentor\Reflection\DocBlockFactory;
+use PHPUnit\Framework\TestCase;
+use LanguageServer\ClientHandler;
+use LanguageServer\Protocol\Message;
+use AdvancedJsonRpc;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use Sabre\Event\Loop;
+use Microsoft\PhpParser;
+
+$frameworksDir = realpath(__DIR__ . '/../../validation/frameworks');
+
+class ValidationTest extends TestCase
```

Comment:
Could you document what the ValidationTests mean compared to the unit tests that work with the `fixtures` folder? Shall all tests be written using the JSON files going forward?